lib/common/ipc_client.c

```diff
@@ -1379,8 +1411,17 @@ pcmk__ipc_is_authentic_process_active(const char *name, uid_t refuid,
         goto bail;
     }
 
-    auth_rc = crm_ipc_is_authentic_process(fd, refuid, refgid, &found_pid,
-                                           &found_uid, &found_gid);
+    auth_rc = pcmk__crm_ipc_is_authentic_process(c, fd, refuid, refgid, &found_pid,
+                                                 &found_uid, &found_gid);
+    if (auth_rc == pcmk_rc_ipc_unauthorized) {
+        crm_err("Daemon (IPC %s) effectively blocked with unauthorized"
+                " process %lld (uid: %lld, gid: %lld)",
+                name, (long long) PCMK__SPECIAL_PID_AS_0(found_pid),
+                (long long) found_uid, (long long) found_gid);
+        rc = pcmk_rc_ipc_unauthorized;
+        goto bail;
+    }
+
     if (auth_rc < 0) {
         rc = pcmk_legacy2rc(auth_rc);
```

Comment:
I just realized we need this to be auth_rc != pcmk_rc_ok and drop the legacy2rc call, since auth_rc is now standard code

Reply:
I also found another one at line 833 !